backend/api/settings.py

```
@@ -218,30 +222,40 @@ class ThrottleScopes:
 # Custom logger just to add some of my own custom logging functions
 # We love DRY!!!
 class PlancakeLogger(logging.Logger):
-    _last_email_time = 0
+    def execute_webhook(self, msg, color):
+        if DISCORD_WEBHOOK_URL:
+            try:
+                requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)